packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -270,14 +270,15 @@ Set<Marker> _rawOptionsToInitialMarkers(Map<String, dynamic> rawOptions) {
         if (rawMarker['position'] != null) {
           position = LatLng.fromJson(rawMarker['position']);
         }
-        if (rawMarker['infoWindow'] != null || rawMarker['snippet'] != null) {
-          String title = rawMarker['infoWindow'] != null
-              ? rawMarker['infoWindow']['title']
-              : null;
-          infoWindow = InfoWindow(
-            title: title ?? '',
-            snippet: rawMarker['snippet'] ?? '',
-          );
+        if (rawMarker['infoWindow'] != null) {
+          final title = rawMarker['infoWindow']['title'];
```

Comment:
nits: We prefer to explicitly declare types: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var

So maybe: `final String title = ..`

ditto several other places in the PR.

Reply:
Should we enable the `always_specify_types` analysis option, so this gets enforced automatically? I think it's enabled in Flutter as well (but this is *forbidden* in g3, btw)

Reply:
https://dart-lang.github.io/linter/lints/always_specify_types.html

Reply:
I've tried enabling this in the repo and... it seems we haven't enforced that rule in a while :)